server/controllers/tasks/lib/tasks.ts

```diff
@@ -82,20 +84,29 @@ export function getTasksBySuggestionUri (suggestionUri) {
   return db.getDocsByViewKey<Task>('bySuggestionUriAndState', [ suggestionUri, null ])
 }
 
-export async function getTasksBySuspectUris (suspectUris: EntityUri[], options: TasksQueryOptions = {}) {
+export async function getTasksBySuspectUrisAndType (uris: EntityUri[], types: string[]) {
+  const keys = zip(uris, types)
```

Comment:
I don't think that works for cases where `uris.length !== 1 || types.length !== 1`
```js
zip([uriA, uriB, uriC], ['deduplicate'],)
// => [ [ uriA, 'deduplicate' ], [ uriB, undefined ], [ uriC, undefined ] ]
```

```suggestion
  const keys = types.flatMap(type => uris.map(uri => [ uri, type ]))
```

Reply:
fixed in a2525a9602